Bugfix in Parser clean() method

dhm80 opened the pull request:
- Now only trimming quotation marks only once at beginning and end of line to prevent breaking wanted quotes at the end of a string like `msgid "This is supposed to be in \"quotation marks\""`
- Added tests for bugfix

MAXakaWIZARD commented:
@c9s Sorry, didn't receive any notifications about this PR since you didn't mention me. As I see, you deleted your repo, so I applied your changes and mentioned you in commit message. Thanks!

dhm80 committed:
Bugfix in Parser clean() method
- Now only trimming quotation marks only once at beginning and end of line to prevent breaking wanted quotes at the end of a string like `msgid "This is supposed to be in \"quotation marks\""`
- Added tests for bugfix

MAXakaWIZARD commented:
Thanks @dhm80!
